src/main/java/com/example/petbuddybackend/utils/exception/throweable/general/NotFoundException.java

```diff
@@ -1,4 +1,4 @@
-package com.example.petbuddybackend.utils.exception.throweable;
+package com.example.petbuddybackend.utils.exception.throweable.general;
 
 public class NotFoundException extends RuntimeException {
 
```

Comment:
odnosze sie do ponizej:
moze dac taki napis?
```suggestion
"\"%s\" :: \"%s\" not found"
```
po dwukropku też będzie wiadomo, a nie zawsze bedzie to nazwa, tylko czasami wlasnie ID